python/lsst/ts/aos/utils/m1m3/M1M3LUT.py

```diff
@@ -104,39 +112,50 @@ async def update_lut_force_balance(
         start_time,
         end_time,
     )
+    print("Elevations\n", elevations["actualPosition"].describe())
+    expected_count = interval_ms / 70.0
+    if len(elevations.index) < expected_count:
+        print(
```

Comment:
isn't the elevation data available always?

Reply:
Of course not. If you stop MTMount CSC, there will be no elevation data.